src/pages/[teamName]/[projectName]/configuration.tsx

```diff
@@ -474,6 +479,14 @@ const ConfigurationPage = withError<
             overflow="auto"
           >
             <Card heading="Build settings" id={`build-settings`}>
+              <Alert status="warning" alignItems="flex-start">
+                <AlertIcon mt={0.75} />
+                <Box>
+                  <AlertDescription color="yellow.900">
```

Comment:
This should use a conditional since in darkmode this is no longer readable 
![image](https://user-images.githubusercontent.com/32865577/84007173-4064ca80-a970-11ea-9b4b-27ea4f400de3.png)

Reply:
Oooh good catch - will resolve now!